Modules/Devices/src/Devices.Infrastructure/PushNotifications/DirectPush/PnsConnectorFactory.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Backbone.Modules.Devices.Domain.Aggregates.PushNotifications;
+
+namespace Backbone.Modules.Devices.Infrastructure.PushNotifications.DirectPush;
+public abstract class PnsConnectorFactory
+{
```

Comment:
The two abstract methods CreateForFirebaseCloudMessaging and CreateForApplePushNotificationService are missing (see class diagramm)

Reply:
As the Connectors for Firebase and Apns are not yet implemented, we had decided to just have the CreateFor method, with a throw NotImplementedException for the time being. Has this changed? If so, how should I declare the abstract methods, seeing as I don't have the classes needed. Should they be implemented like this?  `public abstract IPnsConnector CreateForFirebaseCloudMessaging();`

Reply:
You're right. My bad.